components/camera/camera_test.go

```diff
@@ -183,15 +182,11 @@ func TestCameraWithNoProjector(t *testing.T) {
 	_, got := pc.At(0, 0, 0)
 	test.That(t, got, test.ShouldBeTrue)
 
-	img, _, err := camera.ReadImage(
-		gostream.WithMIMETypeHint(context.Background(), rutils.WithLazyMIMEType(rutils.MimeTypePNG)),
-		noProj2)
```

Comment:
camera.ReadImage still exists & should still behave the same way id did right?

If so, can we please not remove the `camera.ReadImage` test & just add the `camera.ImageFromVideoSource` tests?

Reply:
It is being tested all throughout the transform camera.

Also - these two tests aren't doing much, they're testing projector/ no projector functionality, which was removed, and they can  eventually be removed.

Reply:
https://github.com/viamrobotics/rdk/pull/4487/commits/8244dfde3fcc42f5682cf6893a05d97c480e7ac0